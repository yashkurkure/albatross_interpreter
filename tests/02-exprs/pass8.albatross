printint(16-255|149^131&5+165|140|45|147|137-21+240|129&167-241&6&212&236+180+241-111^54&190^187^154-166|16);
printint(69-252-191+140|68&243-93&250-32|130+232-219-174);
printint(54&179|243^41&219+199+73&17-174-113|107^115^226-205&92-13&236^113+97^40+98);
printint(30+141&235&229-69^164-108+156&86^51);
printint(61&16+152|68|218-27&255^242-120&249-10-98|41-7&26-202|62^180-252|59|43&160+185-50^235+63|18+48);
printint(255+205^8+34&124^134-163-214+120+70&21-86|84&209|96-89+204);
printint(191|149+53+79&5^243-234-150|139&74+82|170|214^64);
printint(106&82^171&247&61-90^72|75-101-139|136^210|152^180|81+195&86&1-20&142);
printint(20|244&202-139-164&77|215&120+165^28|232+189-152^241^215^255-198|47+142+144|151&132);
printint(108);
return 0;
