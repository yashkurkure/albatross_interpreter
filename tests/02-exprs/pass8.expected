-1-9247199-1-257-1-1479108
